doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+.. _plutus_tools_component_descriptions:
+
+Plutus tools component descriptions
+=================================================
+
+The Plutus team has provided a set of “Tools”---components, libraries and 
+executables---for external developers to use. They are located in the 
+`plutus-apps <https://github.com/input-output-hk/plutus-apps>`_ repository. 
+
+For each tool listed below, we have indicated its specific location within 
+plutus-apps, provided a short description, indicated its dependencies, and given 
+a brief statement about its state of development to give you some context. 
+
+1. Plutus script utils
+--------------------------
+
+`Plutus script utils <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-script-utils>`_ 
+is a utility library for helping users write Plutus scripts that are to be used on-chain. 
+Currently, this is a “kitchen sink” library containing code that was moved out of 
+plutus-ledger-api. 
+
+It provides: 
+
+   * hashing functions for Datums, Redeemers and Plutus scripts for any Plutus language version. 
+   * functionality for wrapping the untyped Plutus script with a typed interface. 
+   * utility functions for working with the ScriptContext of a Plutus Script. 
+
++--------------+---------------------------------------------------------------+
+| Plutus Script Utils                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-script-utils``                           |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress with an unstable interface.                  |
++--------------+---------------------------------------------------------------+
+
+2. Marconi
```

Comment:
can't we get automatic section numbering or somethign?

Reply:
I bet we can use some automatic numbering here. I'll have a look at that.

Reply:
And that's an interesting suggestion about `README.md` files in each package directory, etc. If not for this very next iteration, I would like to head in that direction. Seems logical!

Reply:
The automatic numbering doesn't work with lines separated like these headings are. It appears to only work for a series of lines that are together. I've tried it a few ways without success. But doing it manually is no big time sink and it works. 

I totally agree with your suggestion about doing README.md files. I think the first stage is to get this overview page finished. The next stage would be to create or update as many README files as needed in the Tools repo to address all the logical components referenced on this page. Once that's completed, this page could be updated to link to those README files.